src/containers/EntryPage.js

```diff
@@ -159,18 +166,20 @@ class EntryPage extends React.Component {
 }
 
 function mapStateToProps(state, ownProps) {
-  const { collections, entryDraft } = state;
+  const { collections, entryDraft, mediaLibrary } = state;
   const slug = ownProps.match.params.slug;
   const collection = collections.get(ownProps.match.params.name);
-  const newEntry = ownProps.newRecord === true;
+  const newEntry = ownProps.match.newRecord === true;
```

Comment:
This should be reverted, this is breaking new entry creation.

Reply:
Fixed - thanks for finding that.